src/apps/loyalty/containers/forgot_password/browser.tsx

```diff
@@ -5,6 +5,9 @@ import ForgotPassword from "./index"
 const submitUrl = require("sharify").data.SUBMIT_URL
 const appToken = require("sharify").data.APP_TOKEN
 
+import { initAnalytics } from "../../analytics"
```

Comment:
Here's my issue- not every one of our pages has an entrypoint for any JS. Some just are statically rendered server-side.

I sort of want to be able to bundle the `initAnalytics()` stuff into some sort of common JS thats always included on every page. For now, i just dropped these directly here to get something working.